runtime/vm/createramclass.cpp

```diff
@@ -1127,8 +1127,9 @@ fillJITVTableSlot(J9VMThread *vmStruct, UDATA *currentSlot, J9Method *currentMet
 {
 	J9JITConfig *jitConfig = vmStruct->javaVM->jitConfig;
 	UDATA frameBuilder;
-	if (((UDATA)currentMethod->extra & J9_STARTPC_NOT_TRANSLATED) != J9_STARTPC_NOT_TRANSLATED) {
-		frameBuilder = (UDATA)currentMethod->extra;
+	UDATA extra = (UDATA)currentMethod->extra;
+	if ((extra & J9_STARTPC_NOT_TRANSLATED) != J9_STARTPC_NOT_TRANSLATED) {
+		frameBuilder = extra;
```

Comment:
We don't need to introduce a new local for `extra` here.  Better to reuse `frameBuilder`:
```suggestion
	UDATA frameBuilder = (UDATA)currentMethod->extra;
	if ((frameBuilder & J9_STARTPC_NOT_TRANSLATED) != J9_STARTPC_NOT_TRANSLATED) {
```

Note also that this is during class creation so the ramClass hasn't "escaped" to anyone else who can modify it yet